### PR TITLE
Haptic stimulator support, first version

### PR DESCRIPTION
**Pull request by develancer**
First version. Criticism highly appreciated! @mroja @mdovgialo @achabuda

### CODE REVIEW

File: psychopy/app/builder/components/haptic.py [hunk 1]
```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf-8 -*-
+# author: Piotr Różański
+
+from _base import *
```
Comment (mroja) [inline psychopy/app/builder/components/haptic.py]: Do not use global imports.

Comment (develancer) [inline psychopy/app/builder/components/haptic.py]: This one has been copied directly from appliance.py.

Comment (mroja) [inline psychopy/app/builder/components/haptic.py]: Someone else using bad practices doesn't mean you also have to.

Comment (develancer) [inline psychopy/app/builder/components/haptic.py]: Naturally. Will fix.

**Pull request merged**